frontend/src/components/createProjectForm.tsx

```diff
@@ -0,0 +1,367 @@
+        setCoaches(coaches.filter((value) => value !== undefined));
+    }
+
+    function handleDeleteSkill(index: number) {
+        delete skills[index];
+        delete skillInfos[index];
+        setSkills(skills.filter((value) => value !== undefined));
+        setSkillInfos(skillInfos.filter((value) => value !== undefined));
+    }
+
+    async function handleSubmit(submitValues: FormSubmitValues) {
+        const coachURLs: string[] = [];
+        for (let coach of coaches) {
+            coachURLs.push(
+                // We know there will always be a user with this callname,
+                // as every option of the Select contains a value that originates from the users-array
+                users.find((item) => item.callName === coach)!._links.self.href
+            );
+        }
+
+        const createValues: ProjectCreationValues = {
+            name: submitValues.name,
+            info: submitValues.info,
+            versionManagement: submitValues.versionManagement,
+            partnerName: submitValues.partnerName,
+            partnerWebsite: submitValues.partnerWebsite,
+            skills: skills,
+            creator: "",
+            edition: apiPaths.editions + "/" + extractIdFromApiEntityUrl(edition!._links.self.href),
+            skillInfos: skillInfos,
+            goals: goals,
+            coaches: coachURLs,
+        };
+
+        props.submitHandler(createValues, router);
+    }
+
+    function initialize() {
+        setSelectedCoach(users[0].callName);
+        setSelectedSkill(skillTypes[0].name);
+    }
+
+    return (
+        <div className={styles.create_project_box} data-testid="create-project-form" onLoad={initialize}>
+            <Formik
+                initialValues={{
+                    name: "",
+                    info: "",
+                    versionManagement: "",
+                    partnerName: "",
+                    partnerWebsite: "",
+                }}
+                onSubmit={handleSubmit}
+            >
+                <Form>
+                    <h2>{capitalize(t("create project"))}</h2>
+                    <Field
+                        className="form-control mb-2"
+                        label={capitalize(t("project name"))}
+                        name="name"
+                        data-testid="projectname-input"
+                        placeholder={capitalize(t("project name placeholder"))}
+                        required
+                    />
+                    <Field
+                        className="form-control mb-2"
+                        label={capitalize(t("enter project info"))}
+                        name="info"
+                        data-testid="projectinfo-input"
+                        placeholder={capitalize(t("project info placeholder"))}
+                    />
+                    {goals.map((goal: string, index: number) => (
+                        <Row key={index}>
+                            <Col>{goal}</Col>
```

Comment:
Not sure, is this a hard coded string?

Reply:
No, it's just every selected goal being mapped to an element that can be displayed.